app/models/course_team.rb

```diff
@@ -65,9 +65,20 @@ def self.export_fields(options)
   def add_member(user, _id = nil)
     raise "The user \"#{user.name}\" is already a member of the team, \"#{name}\"" if user?(user)
 
-    t_user = TeamsUser.create(user_id: user.id, team_id: id)
+    add_participant(parent_id, user)
```

Comment:
This code should not be present. If we have participant_id in our table, we should directly get the participant's information and not through user_id.